core/idl_types.h

```diff
@@ -20,5 +20,9 @@
 #include "core/idl_base.h"
 
 struct IDLDouble final : public IDLBaseHelper<double> {};
+struct IDLFloat final : public IDLBaseHelper<float> {};
+struct IDLInt32 final : public IDLBaseHelper<int32_t> {};
```

Comment:
There is no int32 type in WebIDL.
Please see https://heycam.github.io/webidl/#idl-types.